packages/mdc-drawer/persistent/mdc-persistent-drawer.scss

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "../mixins";
+@import "../slidable/mixins";
+@import "../slidable/variables";
+
+$mdc-persistent-drawer-width: 240px;
+
+.mdc-persistent-drawer {
+  @include mdc-drawer-base_;
+  width: 0;
+
+  &__drawer {
+    @include mdc-theme-prop(background, background);
+    @include mdc-rtl-reflexive-box(border, right, 1px solid #e4e4e4);
+    @include mdc-rtl-reflexive-position(left, 0);
+
+    @include mdc-theme-dark {
+      background: #212121;
+
+      @include mdc-theme-prop(color, text-primary-on-dark);
+      @include mdc-rtl-reflexive-box(border, right, 1px solid rgba(white, .12));
+    }
+
+    @include mdc-slideable-drawer;
+
+    @include mdc-rtl(".mdc-persistent-drawer") {
+        @include mdc-slideable-drawer-rtl;
+    }
+
+    display: inline-flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    overflow: hidden;
+    touch-action: none;
+    width: $mdc-persistent-drawer-width;
+  }
+
+  &--animating {
+    transition: mdc-animation-enter(width, $mdc-slidable-drawer-transition-time);
```

Comment:
Hmm, you definitely don't want to be animating `width` here if at all possible, as it has [severe performance implications](https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/), especially considering that this will tend to have implications on how the entire page is laid out. I'd be okay with a jump-cut here. Otherwise, I'd look into using something like the [FLIP](https://aerotwist.com/blog/flip-your-animations/) model to see if we can achieve the same affect using transforms.